husky_gazebo/launch/controller_utils.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Robot state publisher -->
+  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <param name="tf_prefix" type="string" value="" />
+  </node>
+
+ <!-- Joint state publisher -->
+  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
```

Comment:
This is the core of my argument for moving from args to optenvs— they're naturally global, so you get them across launchers, xacro, etc, without having to explicitly pass them everywhere.

See: http://wiki.ros.org/jackal_description